packages/back-end/src/controllers/person.ts

```diff
@@ -0,0 +1,11 @@
+import { Request, Response, Express } from 'express';
+import { DatabaseService } from '../entities/database';
+import { CreateAppOptions } from '..';
+
+export const createPerson =
+  ({ databaseService, personCreationService }: CreateAppOptions) =>
+  async (req: Request, res: Response) => {
+    const person = req.body;
```

Comment:
We should check that the data is as expected, and doesn't have extra fields. TS doesn't validate the body so that's a security concern